test-lib.sh

```diff
@@ -773,4 +773,16 @@ ct_check_image_availability() {
   fi
 }
 
+# ct_check_latest_imagestreams
+# -----------------------------
+# Check if the latest version present in Makefile in the variable VERSIONS
+# is present in all imagestreams.
+# Also the latest tag in the imagestreams has to contain the latest version
+ct_check_latest_imagestreams() {
+    local latest_version=
+
+    latest_version=$(grep 'VERSIONS' Makefile | rev | cut -d ' ' -f 1 | rev )
+    python3 ./common/check_imagestreams.py "$latest_version"
```

Comment:
`./common` will work only when we are in the git repo root. It might be enough for the default case (`make check`), but will not work when I run `./run` from the `tests/` dir. I'm not sure what is the best solution here, but there is already `$(dirname "$(readlink -f "$0")")` used below, so maybe we can use the same trick and call the `check_imagestreams.py` from the same directory as this `test-lib.sh` is in..